pkgs/top-level/all-packages.nix

```diff
@@ -5201,12 +5201,20 @@ let
 
   php = php56;
 
-  phpPackages = recurseIntoAttrs (callPackage ./php-packages.nix {});
+  phpPackages = php56Packages;
 
   php55Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
     php = php55;
   });
 
+  php56Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
+    php = php56;
+  });
+
+  php70Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
```

Comment:
Please remove `recurseIntoAttrs`. The PHP packages don't disambiguate their `name` attribute depending on the PHP version, AFAIK, so having multiple PHP package sets will confuse `nix-env`.